tools/dotnet-linker/Compat.cs

```diff
@@ -44,26 +44,6 @@ public Application (LinkerConfiguration configuration)
 			}
 		}
 
-		public void SelectRegistrar ()
-		{
-			if (Registrar == RegistrarMode.Default) {
-				if (IsDeviceBuild) {
-					// mobile device builds use the static registrar by default
-					Registrar = RegistrarMode.Static;
-				} else if (Platform == ApplePlatform.MacOSX && !EnableDebug) {
-					// release macOS builds use the static registrar by default
-					Registrar = RegistrarMode.Static;
-				} else if (!AreAnyAssembliesTrimmed && IsDefaultMarshalManagedExceptionMode) {
-					// Otherwise use the partial static registrar if we can
-					Registrar = RegistrarMode.PartialStatic;
-				} else {
-					// Last option is the dynamic registrar
-					Registrar = RegistrarMode.Dynamic;
-				}
-			}
-			Driver.Log (1, $"Registrar mode: {Registrar}");
```

Comment:
Should we add logging in the callers for better diagnostics?

Reply:
It's already logged:

https://github.com/xamarin/xamarin-macios/blob/fe695d30bd939dddc26e9d3e6fb278039f02f979/tools/dotnet-linker/LinkerConfiguration.cs#L465